src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts

```diff
@@ -90,6 +90,12 @@ export class DataChartComponent implements OnInit, OnDestroy {
     this.alwaysShowCalendars = true;
   }
 
+  @Input()
+  set globalSelectedFilters(val: any) {
```

Comment:
Put this functionality in the method where the filter change event is subscribed

Reply:
resolved